python/baseline/remote.py

```diff
@@ -40,17 +42,24 @@ def predict(self, examples):
         :param examples: The input examples
         :return: The outcomes
         """
-        import requests
+        import httplib, urllib
+        import json
 
         valid_example = all(k in examples for k in self.input_keys)
         if not valid_example:
             raise ValueError("should have keys: " + ",".join(self.input_keys))
 
         v_str = '/versions/{}'.format(self.version) if self.version is not None else ''
         url = '{}/v1/models/{}{}:predict'.format(self.remote, self.name, v_str)
+        path = '/v1/models/{}{}:predict'.format(self.name, v_str)
         request = self.create_request(examples)
-        outcomes_list = requests.post(url, json=request)
-        outcomes_list = outcomes_list.json()
+        headers = {'Content-type': 'application/json' }
+        
+        _, hostname, port = self.remote.split(':')
+        conn = httplib.HTTPConnection(hostname.replace('//', ''), port)
```

Comment:
```suggestion
        conn = HTTPConnection(hostname.replace('//', ''), port)
```

This will make the change to use six easier

Reply:
still please move import up to the top of the file